cdm/src/main/java/ucar/nc2/constants/FeatureType.java

```diff
@@ -52,6 +53,8 @@
 
   ANY_POINT,  // Any of the point types
   POINT,      // unconnected points
+  LINE,		  // two or more connected points
+  POLYGON,	  // three or more connected lines which enclose an area

```

Comment:
It's possible we will need to expand this to include Multiline and Multipolygon in the future. I think this is a good for now though.